From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 25 Aug 2023 15:23:35 +0200
Subject: iconGrid: Export zoomOutActor()

It's used in the search view when launching a search in the
corresponding app.

https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/6942

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2911>
(cherry picked from commit 4386bc49d9f650ecfa4c079a0ceb10144df2672a)
---
 js/ui/iconGrid.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/iconGrid.js b/js/ui/iconGrid.js
index 1b96235..e7bb62f 100644
--- a/js/ui/iconGrid.js
+++ b/js/ui/iconGrid.js
@@ -178,7 +178,7 @@ class BaseIcon extends Shell.SquareBin {
 /**
  * @param {Clutter.Actor} actor
  */
-function zoomOutActor(actor) {
+export function zoomOutActor(actor) {
     let [x, y] = actor.get_transformed_position();
     zoomOutActorAtPos(actor, x, y);
 }
